src/mpi/coll/allreduce/allreduce_intra_ring.c

```diff
@@ -126,7 +126,7 @@ int MPIR_Allreduce_intra_ring(const void *sendbuf, void *recvbuf, MPI_Aint count
 
         MPL_DBG_MSG_FMT(MPIR_DBG_COLL, VERBOSE,
                         (MPL_DBG_FDEST,
-                         "displs[recv_rank:%d]:%d, cnts[recv_rank:%d, displs[send_rank:%d]:%d, cnts[send_rank:%d]:%d]:%d ",
+                         "displs[recv_rank:%d]:%ld, cnts[recv_rank:%d]:%ld, displs[send_rank:%d]:%ld, cnts[send_rank:%d]:%ld ",
                          recv_rank, displs[recv_rank], recv_rank, cnts[recv_rank], send_rank,
                          displs[send_rank], send_rank, cnts[send_rank]));
```

Comment:
`%ld` is not the correct format for `MPI_Aint`, for example on 32-bit systems, or if `MPI_Aint` is long long.
Can we just remove these debug statements? It is not worth the effort maintaining them.

Reply:
@hzhou Do you want to remove this debug message in your active PR https://github.com/pmodels/mpich/pull/5940?  As they fall under the same theme.

Reply:
I prefer your team does it :)

Reply:
Okay, I can make that change in this PR then. :)